src/aiortc/codecs/keypointcodec.py

```diff
@@ -192,8 +192,16 @@ def _convert(data: bytes, width: int) -> bytes:
 
     def decode(self, encoded_frame: JitterFrame) -> List[KeypointsFrame]:
         keypoint_str = encoded_frame.data
-        keypoint_dict = custom_decode(keypoint_str)
-        keypoint_dict['pts'] = 5
+
+        if NUM_JACOBIAN_BITS == -1:
+            keypoint_info_struct = KeypointInfo()
+            keypoint_info_struct.ParseFromString(keypoint_str)
+            assert(keypoint_info_struct.IsInitialized())
+            keypoint_dict = keypoint_struct_to_dict(keypoint_info_struct)
+        else:
+            keypoint_dict = custom_decode(keypoint_str)
+            keypoint_dict['pts'] = 5
```

Comment:
Why 5?

Reply:
Can we have a global constant?

Reply:
Good point - This is just a hack for now to set it to something explicit - it doesn't get used, so I decided to make it something random